src/components/subscriptions/data/service.js

```diff
@@ -114,10 +114,10 @@ class LicenseManagerApiService {
     return LicenseManagerApiService.apiClient().get(url);
   }
 
-  static fetchSubscriptionUsers(subscriptionUUID, options) {
+  static fetchSubscriptionUsers(subscriptionUUID, options, pageSize = PAGE_SIZE) {
```

Comment:
nit: I think it makes sense to keep all the options for the endpoint in the `options` function argument rather than introducing a separate `pageSize` function argument here, especially if this new argument isn't getting used yet.

By keeping everything together in a single `options` object, you could pass everything straight to query parameters while still allowing an overridable default page size option:

```jsx
const queryParams = {
  page_size: PAGE_SIZE,
  ...options,
};
```

```jsx
fetchSubscriptionUsers(subscriptionUUID, { page: 1, page_size: 50 });
```